### PR TITLE
Integrations API doc refactoring

### PR DESCRIPTION
**Pull request by l0k0ms**
### What does this PR do?
This PR breaks down the current integration API documentation into 5 new one:

* [AWS integration](https://docs-staging.datadoghq.com/gus/api-integrations-refactoring/api/?lang=bash#aws-integration)
* [Azure integration](https://docs-staging.datadoghq.com/gus/api-integrations-refactoring/api/?lang=bash#azure-integration)
* [GCP integration](https://docs-staging.datadoghq.com/gus/api-integrations-refactoring/api/?lang=bash#gcp-integration)
* [PagerDuty integration](https://docs-staging.datadoghq.com/gus/api-integrations-refactoring/api/?lang=bash#pagerduty-integration)
* [Slack integration](https://docs-staging.datadoghq.com/gus/api-integrations-refactoring/api/?lang=bash#slack-integration)
* [Webhooks integration](https://docs-staging.datadoghq.com/gus/api-integrations-refactoring/api/?lang=bash#webhooks-integration)

It also checks and updates all example given in the public documentation and in the postman collection.

### Motivation
Adding new integration example was becoming even more difficult with the current layout that btw didn't follow documentation guidelines + Some example were just plain wrong.

### CODE REVIEW

File: content/en/api/integrations_gcp/gcp_create.md [hunk 1]
```diff
@@ -1,31 +1,29 @@
...
-[Read more about Datadog-Google Cloud Platform integration][1].
+Create a Datadog-Google Cloud Platform integration.
 
-**Note**: 
+**Note**:
 
-* Using the `POST` method updates your integration configuration by **adding** your new configuration to the existing one in your Datadog organization. 
-
-* Using the `PUT` method updates the existing integration configuration in your Datadog organization.
+* Using the `POST` method updates your integration configuration by **adding** your new configuration to the existing one in your Datadog organization.
```
Comment (apigirl) [inline content/en/api/integrations_gcp/gcp_create.md]: I'm a bit unclear here - does this add an additional config file, or does it combine your existing config with the new one?

File: content/en/api/integrations_gcp/gcp_create.md [hunk 2]
```diff
@@ -1,31 +1,29 @@
...
 
-All of the following fields' values are provided by the JSON service account key file created in the [GCP Console for service accounts][2]; Refer to the [Datadog-Google Cloud Platform integration installation instructions][3] to see how to generate one for your organization. 
-For further references, consult the [Google Cloud service account documentation][4].
+All of the following fields' values are provided by the JSON service account key file created in the [GCP Console for service accounts][1]; Refer to the [Datadog-Google Cloud Platform integration installation instructions][2] to see how to generate one for your organization.
+For further references, consult the [Google Cloud service account documentation][3].
 
-* **`type`** [*required*]:  
+* **`type`** [*required*]:
 
     Should be `service_account`, it can be found in your JSON service account key.
```
Comment (apigirl) [inline content/en/api/integrations_gcp/gcp_create.md]:
```suggestion
    The value for `service_account` found in your JSON service account key.
```

File: content/en/api/integrations_gcp/gcp_host_filters.md [hunk 3]
```diff
@@ -0,0 +1,25 @@
+---
+title: Update a GCP integration host filters
+type: apicontent
+order: 17.5
+external_redirect: /api/#update-a-gcp-integration-host-filters
+---
+
+## Update a GCP integration host filters
+
+Updates the defined list of host filters for a given Datadog-GCP integration.
```
Comment (apigirl) [inline content/en/api/integrations_gcp/gcp_host_filters.md]:
```suggestion
Update the defined list of host filters for a given Datadog-GCP integration.
```

File: content/en/api/integrations_pagerduty/pagerduty.md [hunk 4]
```diff
@@ -0,0 +1,13 @@
...
+title: PagerDuty Integration
+type: apicontent
+order: 18
+external_redirect: /api/#pagerduty-integration
+---
+
+## PagerDuty Integration
+
+Configure your Datadog-PagerDuty integration directly through the Datadog API.
+[Read more about Datadog-PagerDuty integration][1].
```
Comment (apigirl) [inline content/en/api/integrations_pagerduty/pagerduty.md]:
```suggestion
For more informations, see the [PagerDuty integration page][1].
```

File: content/en/api/integrations_pagerduty/pagerduty_add_services.md [hunk 5]
```diff
@@ -0,0 +1,27 @@
...
+    Array of PagerDuty service objects. [Learn how to configure you Datadog service with PagerDuty documentation][1]. A PagerDuty service object is composed by:
+
+    * **`service_name`** [*required*]:
+        Your Service name in PagerDuty.
+
+    * **`service_key`** [*required*]:
+        Your Service name associated service key in Pagerduty.
+
+* **`schedules`** :
+    Array of your schedule URLs, e.g:
```
Comment (apigirl) [inline content/en/api/integrations_pagerduty/pagerduty_add_services.md]:
```suggestion
    The array of your schedule URLs:
```

File: content/en/api/integrations_pagerduty/pagerduty_add_services.md [hunk 6]
```diff
@@ -0,0 +1,27 @@
...
+---
+
+## Add new services and schedules
+
+Add new services and schedules to your Datadog-PagerDuty integration.
+
+##### Arguments
+
+* **`services`** :
+    Array of PagerDuty service objects. [Learn how to configure you Datadog service with PagerDuty documentation][1]. A PagerDuty service object is composed by:
```
Comment (apigirl) [inline content/en/api/integrations_pagerduty/pagerduty_add_services.md]:
```suggestion
    The array of PagerDuty service objects. Learn how to configure your Datadog service with the [PagerDuty documentation][1]. A PagerDuty service object is composed by:
```

File: content/en/api/integrations_pagerduty/pagerduty_create.md [hunk 7]
```diff
@@ -0,0 +1,35 @@
...
+## Create a PagerDuty integration
+
+Create a new Datadog-PagerDuty integration.
+
+**Note**: All arguments are required when creating (`PUT`) a new PagerDuty configuration.
+
+##### ARGUMENTS
+
+* **`services`** :
+    Array of PagerDuty service objects. [Learn how to configure you Datadog service with PagerDuty documentation][1]. A PagerDuty service object is composed by:
```
Comment (apigirl) [inline content/en/api/integrations_pagerduty/pagerduty_create.md]:
```suggestion
    The array of PagerDuty service objects. Learn how to configure your Datadog service with the [PagerDuty documentation][1]. A PagerDuty service object is composed by:
```

File: content/en/api/integrations_pagerduty/pagerduty_create_code.md [hunk 8]
```diff
@@ -0,0 +1,16 @@
+---
+title: Create a PagerDuty integration
+type: apicode
+order: 18.2
+external_redirect: /api/#create-a-pagerduty-integration
+---
+
+##### Signature
+`PUT https://api.datadoghq.com/api/v1/integration/pagerduty`
```
Comment (apigirl) [inline content/en/api/integrations_pagerduty/pagerduty_create_code.md]: shouldn't this be a POST?

File: content/en/api/integrations_pagerduty/pagerduty_get.md [hunk 9]
```diff
@@ -0,0 +1,14 @@
+---
+title: Get a PagerDuty integration
+type: apicontent
+order: 18.1
+external_redirect: /api/#get-a-pagerduty-integration
+---
+
+## Get a PagerDuty integration
+
+Get all information about your Datadog-PagerDuty integration
```
Comment (apigirl) [inline content/en/api/integrations_pagerduty/pagerduty_get.md]:
```suggestion
Get all information about your Datadog-PagerDuty integration.
```

File: content/en/api/integrations_pagerduty/code_snippets/pagerduty_create.sh [hunk 10]
```diff
@@ -0,0 +1,24 @@
+#!/bin/sh
+# Make sure you replace the <DD_API_KEY> and <DD_APP_KEY> key below
+# with the ones for your account
+
+api_key=<DD_API_KEY>
+app_key=<DD_APP_KEY>
+
+curl -v -X PUT -H "Content-type: application/json" \
```
Comment (apigirl) [inline content/en/api/integrations_pagerduty/code_snippets/pagerduty_create.sh]: I would have guessed a create is a POST?

File: content/en/api/integrations_slack/slack.md [hunk 11]
```diff
@@ -0,0 +1,13 @@
...
+title: Slack Integration
+type: apicontent
+order: 19
+external_redirect: /api/#slack-integration
+---
+
+## Slack Integration
+
+Configure your Datadog-Slack integration directly through Datadog API.
+[Read more about Datadog-Slack integration][1].
```
Comment (apigirl) [inline content/en/api/integrations_slack/slack.md]:
```suggestion
For more information about the Datadog-Slack integration, see the [integration page][1].
```

File: content/en/api/integrations_slack/slack_add_channels.md [hunk 12]
```diff
@@ -0,0 +1,26 @@
...
+---
+
+## Add channels to Slack integration
+
+Add channels to your Datadog-Slack integration.
+
+##### ARGUMENTS
+
+* **`channels`** [*required*]:
+    Array of slack channel objects to post to. A slack channel object is composed by:
```
Comment (apigirl) [inline content/en/api/integrations_slack/slack_add_channels.md]:
```suggestion
    The array of slack channel objects to post to. A slack channel object is composed by:
```

File: content/en/api/integrations_slack/slack_delete.md [hunk 13]
```diff
@@ -0,0 +1,14 @@
+---
+title: Delete a Slack integration
+type: apicontent
+order: 19.4
+external_redirect: /api/#delete-a-slack-integration
+---
+
+## Delete a Slack integration
+
+Delete a Datadog-Slack integration
```
Comment (apigirl) [inline content/en/api/integrations_slack/slack_delete.md]:
```suggestion
Delete a Datadog-Slack integration.
```

File: content/en/api/integrations_webhook/webhooks.md [hunk 14]
```diff
@@ -0,0 +1,13 @@
+---
+title: Webhooks Integration
+type: apicontent
+order: 20
+external_redirect: /api/#webhooks-integration
+---
+
+## Webhooks Integration
+
+Configure your Datadog-Webhooks integration directly through Datadog API.
```
Comment (apigirl) [inline content/en/api/integrations_webhook/webhooks.md]:
```suggestion
Configure your Datadog-Webhooks integration directly through the Datadog API.
```

File: content/en/api/integrations_webhook/webhooks.md [hunk 15]
```diff
@@ -0,0 +1,13 @@
...
+[Read more about Datadog-Webhooks integration][1].
```
Comment (apigirl) [inline content/en/api/integrations_webhook/webhooks.md]:
```suggestion
For more information about the Datadog-Webhooks integration, see the [integration page][1].
```

File: content/en/api/integrations_gcp/gcp_create.md (hunk 1 above)
Comment (l0k0ms) [reply content/en/api/integrations_gcp/gcp_create.md]: Yup this add a new configuration file, the `PUT` method would combine the config pushed with the existing configuration if any.

File: content/en/api/integrations_pagerduty/pagerduty_create_code.md (hunk 8 above)
Comment (l0k0ms) [reply content/en/api/integrations_pagerduty/pagerduty_create_code.md]: This is what i thought too.. but no this is a `PUT` after testing idk why.

**Pull request merged**